modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,211 @@
+
+== Cancel a migration
+
+You can cancel a topic migration by running the command:
+
+[tabs]
+======
+rpk::
++
+--
+```
+rpk cluster storage cancel-mount <migration-id>
+```
+--
+
+Admin API::
++
+--
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=cancel
+```
+--
+======
+
+You cannot cancel migrations in the following <<monitor-progress,states>>:
+
+- `planned` (but you may still xref:api:ROOT:admin-api.adoc#delete-/v1/migrations/-id-[delete] a planned migration)
+- `cut_over`
+- `finished`
+- `canceling`
+- `cancelled`
+
+== Monitor progress
+
+[tabs]
+======
+rpk::
++
+--
+To list topic migrations, run the command:
+
+```
+rpk cluster storage list-mount
+```
+--
+
+Admin API::
++
+--
+Issue a GET request to the `/migrations` endpoint to view the state of topic migrations:
+
+```
+curl http://localhost:9644/v1/migrations 
+```
+--
+======
+
+The response returns the IDs and state of existing migrations:
+
+|===
+| Migration state | Outbound migration (unmount) | Inbound migration (mount)
+
+| `planned`
+2+| Redpanda validates the migration definition.
+
+
+| `preparing`
+| Redpanda copies all possible topic data, including topic manifests, to the destination bucket or container in object storage.
+| Redpanda recreates the topics in a disabled state in the target cluster. The cluster allocates partitions but does not add log segments yet. Topic metadata is populated from the topic manifests found in object storage.
+
+| `prepared` 
+| The migration is ready to execute. In this state, the cluster still accepts client reads and writes for the topics.
+| Topics exist in the cluster but clients do not yet have access to consume or produce.
+
+| `executing` 
+| The cluster rejects client reads and writes for the topics. Redpanda uploads any remaining topic data that has not yet been copied to object storage. Uncommitted transactions involving the topic are aborted.
+| The target cluster checks that the topic to be mounted has not already been mounted in any cluster.
+
+| `executed` 
+| All unmounted topic data from the cluster is available in object storage.
+| 
+
+| `cut_over`
+| Redpanda deletes topic metadata from the cluster, and marks the data in object storage as available for mount operations.
+| The topic data in object storage is no longer available to mount to any clusters.
+
+| `finished`
+| The migration is complete and then deleted. 
+| The migration is complete and then deleted. The target cluster starts to handle produce and consume workloads.
+
+| `canceling`
+2+| Redpanda is in the process of canceling the migration.
+
+
+| `cancelled`
+2+| The migration is canceled.
+
+|===
+
+== Troubleshoot
+
+If Redpanda encounters errors during mounting and unmounting, it retries the operation indefinitely. If you experience issues, check this list:
+
+* Do not attempt to mount a topic that does not exist in object storage.
+* Do not attempt to mount a topic that is already mounted to the current or a different cluster.
```

Comment:
@kbatuigas instead of just saying don't do this, let's explain how things work.  We have a built-in guardrail to prevent a topic from being mounted to two clusters at once to prevent multiple clusters writing to the same location in cloud storage and corrupting the topic. this is a feature.

cc @andrwng  for commentary

Reply:
@mattschumpert @andrwng for my understanding - are target clusters writing to object storage during the mount?